src/DetailsView/reports/components/outcome-summary-bar.tsx

```diff
@@ -1,33 +1,37 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+import { kebabCase } from 'lodash';
 import * as React from 'react';
 
 import { NamedSFC } from '../../../common/react/named-sfc';
-import { OutcomeIcon } from './outcome-icon';
-import { allRequirementOutcomeTypes, RequirementOutcomeStats } from './requirement-outcome-type';
+import { outcomeIconMap, outcomeIconMapInverted, OutcomeStats, OutcomeType, outcomeTypeSemantics, OutcomeUnits } from './outcome-type';
 
-type OutcomeUnits = 'percentage' | 'requirements';
-
-const outcomeText = {
-    pass: 'Passed',
-    incomplete: 'Incomplete',
-    fail: 'Failed',
+export type OutcomeSummaryBarProps = {
+    outcomeStats: Partial<OutcomeStats>;
+    allOutcomeTypes: OutcomeType[];
+    units?: OutcomeUnits;
+    inverted?: boolean;
```

Comment:
may we rename this to 'iconStyle' or 'invertedIconStyle'... inverted made me think of order (inverted => reverse order) which is not the case.